alarm/utils.py

```diff
@@ -30,6 +37,36 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
                     f"threshold: {threshold}; "
                     f"threshold broken: {threshold_broken}")
         if threshold_broken:
+            ThresholdBrake.objects.create(threshold=threshold)
             return True
 
     return False
+
+
+def check_call_status(call, message):
+    if call.status == 'completed':
+        # Sleep for 15 minutes
+        time.sleep(900)
+    elif call.status == 'busy' or call.status == 'failed':
+        # Sleep for 1 minute and create a new call
+        time.sleep(60)
+        call = client.calls.create(
+            twiml=message,
+            to=USER_PHONE_NUMBER,
+            from_=PHONE_NUMBER_TWILLIO
+        )
+        check_call_status(call)
+    else:
+        # Try to call the user every minute for 15 minutes
+        for i in range(15):
+            time.sleep(60)
+            call = client.calls.create(
+                twiml=message,
+                to=USER_PHONE_NUMBER,
+                from_=PHONE_NUMBER_TWILLIO
+            )
+            if call.status == 'completed':
+                break
+            elif i == 14:
+                # Maximum attempts reached, exit the loop
+                break
```

Comment:
do we need this elif? when "i" will rich 15 cycle will finished

Reply:
You're right. Changed!